src/js/components/Search.js

```diff
@@ -333,36 +327,34 @@ export default class Search extends Component {
     }
 
     return (
-      <div id="search-drop" className={classes.join(' ')}>
+      <div id="search-drop" className={classes}>
         {contents}
       </div>
     );
   }
 
   render () {
-
-    var classes = this._classes(CLASS_ROOT);
-    if (this.props.size) {
-      classes.push(CLASS_ROOT + "--" + this.props.size);
-    } else if (this.props.large && ! this.props.size) {
-      classes.push(CLASS_ROOT + "--large");
-    }
-    if (this.props.className) {
-      classes.push(this.props.className);
-    }
-    if (this.props.iconAlign) {
-      classes.push(CLASS_ROOT + "--icon-align-" + this.props.iconAlign);
-    }
+    let classes = classnames(
+      CLASS_ROOT,
+      this.props.className,
```

Comment:
Can we put this one at the end of the list? It makes it slightly easier to parse the classes in the DOM when they are grouped according to which component sets them.